Tests/CommonTestHelper.psm1

```diff
@@ -762,6 +762,42 @@ function Exit-DscResourceTestEnvironment
     Restore-TestEnvironment -TestEnvironment $TestEnvironment
 }
 
+<#
+    .SYNOPSIS
+        Retrieves the administrator credential on an AppVeyor machine.
+        The password will be reset so that we know what the password is.
+
+    .NOTES
+        The AppVeyor credential will be cached after the first call to this function so that the
+        password is not reset again if this function is called again.
+#>
+function Get-AppVeyorAdministratorCredential
+{
+    [OutputType([System.Management.Automation.PSCredential])]
+    [CmdletBinding()]
+    param ()
+
+    if ($null -eq $script:appVeyorAdministratorCredential)
```

Comment:
@TravisEz13 It's turned on in #289 where I made the main changes to the CommonTestHelper. You can see it is included on [this line](https://github.com/PowerShell/xPSDesiredStateConfiguration/pull/289/files#diff-81c06b99842ae1751400e8f993ee1114R5) for the changes included in that PR. This exact same function is there and passes StrictMode.

I did not want to include all the changes to the CommonTestHelper here. I only included `Get-AppVeyorAdministratorCredential` here because it is needed to get a valid credential to test running a script under.